cekit/tools.py

```diff
@@ -158,7 +158,18 @@ def __init__(self):
             with open(os_release_path, 'r') as f:
                 content = f.readlines()
 
-            self.os_release = dict(l.strip().split('=') for l in content)
```

Comment:
Is it not possible to do " for l in content if not l.isspace() " ?

Reply:
I didn't know about this one, I can try, sure!

Reply:
Although thinking further, I guess that wouldn't handle line starting with '#' which is what https://www.freedesktop.org/software/systemd/man/os-release.html specifies.